.github/workflows/bsr-push.yml

```diff
@@ -0,0 +1,30 @@
+name: Push to Buf Schema Registry
+
+on:
+  workflow_dispatch
+
+jobs:
+  push-module:
+    runs-on: ubuntu-latest
+    steps:
+      # Run `git checkout`
+      - uses: actions/checkout@v2
+      # Install the `buf` CLI
+      - uses: bufbuild/buf-setup-action@v1
+        with:
+          github_token: ${{ github.token }}
+      # Lint Protobuf sources
+      - uses: bufbuild/buf-lint-action@v1
+        with:
+          input: proto
+      # Run breaking change detection for Protobuf sources against the current `main` branch
+      - uses: bufbuild/buf-breaking-action@v1
+        with:
+          against: https://github.com/acme/weather.git#branch=main,ref=HEAD~1,subdir=proto
+      # Push module to the BSR
+      - uses: bufbuild/buf-push-action@v1
+        with:
+          input: proto
+          buf_token: ${{ secrets.BUF_TOKEN }}
```

Comment:
Did you create the secret in GH action?

Reply:
No, I do not seem to have permissions to add tokens to actions.

Technically we should have a non-personal / security managed BSR user that the token belongs to. But I guess to start with a personal token can do.

Reply:
I've been told the secret has been added.